mirrord/config/src/feature/fs/advanced.rs

```diff
@@ -37,8 +40,22 @@ use crate::{
 /// **Warning**: Specifying the same path in two lists is unsupported and can lead to undefined
 /// behaviour.
 ///
-/// 2. Check our "special list" - we have an internal at compile time list
-/// for different behavior based on patterns to provide better UX.
+/// 2. There are pre-defined exceptions to the set FS mode.
+///   a. Paths that match
+///      [the patterns defined here](https://github.com/metalbear-co/mirrord/tree/latest/mirrord/layer/src/file/filter/read_local_by_default.rs)
+///      are read locally by default.
+///   b. Paths that match
+///      [the patterns defined here](https://github.com/metalbear-co/mirrord/tree/latest/mirrord/layer/src/file/filter/read_remote_by_default.rs)
+///      are read remotely by default when the mode is `localwithoverrides`.
+///   c. Paths that match
+///      [the patterns defined here](https://github.com/metalbear-co/mirrord/tree/latest/mirrord/layer/src/file/filter/not_found_by_default.rs)
+///      under the running user's home directory will be failed to be found by default when the mode
+///      is not `local`.
+/// In order to override that default setting for a path, or a pattern, include it the appropriate
```

Comment:
How do you override the "not found" list? Maybe it's better to add an example of how to override to each list, rather than a general example at the bottom.

Reply:
You have to include a path in another list that specifies what should happen with it - `read_only`/`read_write`/`local`.
I'm afraid if I make this text any longer no one will ever read it.